### PR TITLE
Loosen Flux CUDA bounds

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
cc @MikeInnes 

Hopefully these bounds look about right.

### CODE REVIEW

File: F/Flux/Compat.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ ZipFile = "0.7-0.8"
 
 ["0.10.1-0"]
 AbstractTrees = "0.2-0.3"
-CuArrays = "1.6.0-1"
+CuArrays = "1.4.3-1."
```
Comment (MikeInnes) [inline F/Flux/Compat.toml]: What does the `1.` mean here, as opposed to `1`?

Comment (dhairyagandhi96) [reply F/Flux/Compat.toml]: It was a typo; should be fixed now

Thanks!

Comment (MikeInnes) [reply F/Flux/Compat.toml]: Ah, great.

**Comment by DilumAluthge** [comment]
@dhairyagandhi96 @MikeInnes Is this ready to merge?

**Comment by dhairyagandhi96** [comment]
As long as the format looks reasonable, I think I am good with that

**Comment by MikeInnes** [comment]
Yeah it could do with a sanity check from the experts but LGTM

**Review by DilumAluthge — APPROVED** [review]

**Pull request merged**